content/docs/user-guide/components/reference/terrain/world-renderer.md

```diff
@@ -22,3 +22,42 @@ You must add this component to the *Level* entity, the parent of all entities in
+
+### Mesh Configuration properties
+These properties control the mesh density and blending of terrain LODs (level of detail) and have a very direct impact on vertex performance. The density of the first terrain
+LOD is set by the query resolution in the [Terrain World](/docs/user-guide/components/reference/terrain/world) component, and **First LOD distance** controls how far to render at this
+density. Each subsequent LOD will render for twice the distance at half the resolution all the way until **Mesh render distance** is met. This means that **First LOD distance** actually
+has a much greater impact on total triangle count than the total rendering distance.
+
+| Property | Description | Values | Default |
+| - | - | - | - |
+| Mesh render distance | The distance from the camera that terrain meshes will render in meters. | 1.0 to 100000.0 | 4096.0 |
+| First LOD distance | The distance from the camera that the first LOD renders to. Subsequent LODs will be at double the distance from the previous LOD. | 1.0 to 10000.0 | 128.0 |
+| Continuous LOD (CLOD) | Enables the use of continuous level of detail, which smoothly blends geometry between terrain LODs. | Boolean | true |
+| CLOD Distance | Distance in meters over which the first LOD will blend into the next LOD. Subsequent LOD blend distances will double with each LOD for a consistent visual appearance. <br><br>  {{< note >}} Generally this should be less than about 25% of **First LOD distance**. If set too low, seams may appear in the terrain. {{< /note >}} | 0.0 to 1000.0 | 16.0 |
+
+### Detail Configuration
+These settings impact how far detail materials render, how they fade out, and how they blend with each other. Most of these settings have little impact on performance, however a larger
+render distance here will require more surface data points to be queried as you move around the world and also take more memory.
+
+| Property | Description | Values | Default |
+| - | - | - | - |
+| Height based texture blending | When turned on, detail materials will use the height texture to aid with blending. | Boolean | False |
+| Detail material render distance (m) | The distance from the camera that the detail material will render in meters. | 1.0 to 2048.0 | 512.0 |
+| Detail material fade distance (m) | The distance over which the detail material will fade out into the macro material. | 0.0 to 2048.0 | 64.0 |
+| Detail material scale | The scale at which all detail materials are rendered at. | 0.0001 to 10000.0 | 1.0 |
+
+### Clipmap Configuration
+
+Clipmaps are a stack of baked textures centered at the view and covering different distances at different resolutions. Detail materials and macro materials are baked into 2 sets of clipmaps which will take their place when the terrain is rendered. Using clipmaps will have a substantial gain in performance at the cost of texture memory. Clipmap configuration can be used to find a balance between the performance gain and the memory cost.
```

Comment:
- Wdym by "centered at the view"?
- In my suggested rephrasing, is it accurate to say that the detail and macro materials "replace the clipmaps"?
```suggestion
_Clipmaps_ are a stack of baked textures that are centered at the view. They include baked textures at different distances and at different resolutions. Detail materials and macro materials are baked into two sets of clipmaps. They replace the clipmaps when rendering the terrain. Using clipmaps improves performance, but at the cost of texture memory. You can adjust the clipmap configuration to find a balance between the performance gain and the memory cost. 
```

Reply:
This needed a bit more work, I messaged you directly with a suggestion.